allofplos/plos_corpus.py

```diff
@@ -720,20 +739,22 @@ def create_local_plos_corpus(corpusdir=corpusdir, rm_metadata=True):
         os.remove(metadata_path)
 
 
-def create_test_plos_corpus(corpusdir=corpusdir):
+def create_test_plos_corpus(directory=None):
     """
     Downloads a copy of 10,000 randomly selected PLOS articles by:
-    1) creating corpusdir if it doesn't exist
+    1) creating directory if it doesn't exist
     2) downloading the zip file (defaults to corpus directory)
     3) extracting the individual XML files into the corpus directory
-    :param corpusdir: directory where the corpus is to be downloaded and extracted
+    :param directory: directory where the corpus is to be downloaded and extracted
     :return: None
     """
-    if os.path.isdir(corpusdir) is False:
-        os.mkdir(corpusdir)
+    if directory is None:
+        directory = get_corpus_dir()
+    if os.path.isdir(directory) is False:
```

Comment:
Can be condensed to one line
`os.makedirs(directory, exist_ok=True)`
Applies to other locations as well.

Reply:
fixed